File: 3242-count-elements-with-maximum-frequency/3242-count-elements-with-maximum-frequency.py
class Solution(object):
    def maxFrequencyElements(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        max=0
        n=0
        knum={}
        for i in nums:
            if i in knum.keys():
                knum[i]+=1
            else:
                knum[i]=1
        for (i,j) in knum.items():
            if j>max:
                max=j
                n=0
            if j==max :
                n+=1
        return max*n
            
            
        

File: 3242-count-elements-with-maximum-frequency/3242-count-elements-with-maximum-frequency.c
int maxFrequencyElements(int* nums, int numsSize) {
    int k[100]={0};
    for(int i=0;i<numsSize;i++)k[nums[i]-1]++;
    int max=k[0],n=0;
    for(int i=0;i<100;i++){
        if(k[i]>max){max=k[i];n=0;};
        if(k[i]==max)n++;
    }
    return max*n;
}